clms-angular/src/app/modules/authentication/pages/login/login.component.ts

```diff
@@ -62,6 +64,7 @@ export class LoginComponent {
     this.loginService.login(this.username.value || "", this.password.value || "")
       .pipe(tap(response => this.loadingAmbianceService.loadingAmbianceState = LoadingAmbianceState.NONE))
       .subscribe((response) => {
+        this.clientDataSourceService.refresh();
```

Comment:
Once a user logs in we refresh the clientDataSourceService to fetch client details